### DISCUSSION TITLE
group together aws monorepo version 1 and version 2 sdks

### DISCUSSION THREAD

**Discussion by BusyByte** [renovatebot/renovate#19225] [Request Help] [answered]
We have a repo that has both aws 1 and 2 monorepo prs being opened.
There are predefined group for both of those (aws-java-sdk, aws-java-sdk-v2) but we'd like to group both of these mono-repos together.  This way we can merge them together and reduce the noise.

Are there any examples of how to do this?

**Comment by rarkins** [comment, marked as answer]
You need to create a package rule with matchPackagePatterns + groupName, there are many examples of these features being used in the docs and existing discussions

**Reply by BusyByte** [reply]
@rarkins So it won't conflict with the existing monorepo definitions/groupings just to create a new group with overlapping patterns?

**Reply by rarkins** [reply]
Your own packageRules will essentially _override_ the grouping which comes as part of the monorepo definitions. Last rule wins, if they conflict
